x-pack/plugins/monitoring/server/lib/kibana/get_kibanas.js

```diff
@@ -73,4 +73,6 @@ export function getKibanas(req, kbnIndexPattern, { clusterUuid }) {
       };
     });
   });
+
+  return instances;
```

Comment:
Doesn't look like the changes in this file change the functionality or related to the PR

Reply:
yea I'm not sure what this is either